### PR TITLE
Update/csi 1349 add community csv

### PR DESCRIPTION
**Pull request by oriyarde**
for community OperatorHub.

references:
Community nested: https://github.com/operator-framework/community-operators/blob/master/docs/contributing.md#create-a-bundle
Red Hat flat: https://redhat-connect.gitbook.io/certified-operator-guide/what-if-ive-already-published-a-community-operator/assembling-the-metadata-bundle
Manifest verify tool: https://github.com/operator-framework/operator-courier/#command-line-interface

TODO - integrate more CI tests:
https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md#validate-bundle-image
https://github.com/operator-framework/operator-sdk/blob/v0.16.0/doc/cli/operator-sdk_bundle_validate.md
https://github.com/operator-framework/operator-sdk/blob/v0.16.0/doc/test-framework/scorecard.md#tests-performed

---
This change is [Reviewable](https://reviewable.io/reviews/ibm/ibm-block-csi-operator/67)

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     capabilities: "Basic Install"
     categories: "Storage,Cloud Provider"
     certified: "true"
-    containerImage: ibmcom/ibm-block-csi-operator:1.1.0
+    containerImage: registry.connect.redhat.com/ibm/ibm-block-csi-operator:1.1.0
```
Comment (ArbelNathan) [inline deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml:10]: what is the difference?

File: build/ci/jenkins_pipeline_build [hunk 2]
```diff
@@ -3,10 +3,19 @@ pipeline {
         label 'docker-engine'
     }
     stages {
-        stage ('Run unit tests') {
-            steps {
-                sh 'mkdir -p build/_output/reports && chmod 777 build/_output/reports'
-                sh './build/ci/run_unittests.sh'
+        stage ('Running tests in parallel') {
```
Comment (igorgonibm) [inline build/ci/jenkins_pipeline_build:6]: why is it in parallel?

File: deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml:10]: `ibmcom` is DockerHub (for community) and the second is for Red Hat.
but I shouldn't change it here:
https://github.com/IBM/ibm-block-csi-operator/pull/66
will revert this

File: build/ci/jenkins_pipeline_build (hunk 2 above)
Comment (oriyarde) [reply build/ci/jenkins_pipeline_build:6]: why not?

File: build/ci/Dockerfile.courier [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Copyright IBM Corporation 2019.
```
[the remark this answers is not in the document]
Comment (oriyarde) [reply build/ci/Dockerfile.courier:1]: not sure, but will update it

File: build/ci/jenkins_pipeline_build (hunk 2 above)
Comment (igorgonibm) [reply build/ci/jenkins_pipeline_build:6]: The log of parallel parts can be mixed here?
E.g. can we see in logs/console something like this?
unitest log msg 1
olm validation log 1
unitest log msg 2
olm validation log 2

Comment (oriyarde) [reply build/ci/jenkins_pipeline_build:6]: depends on where you look.
I guess specifically console output will be mixed.
if you prefer to permanently double the build time for a serial log (even when there's no yaml change), I will remove it

Comment (igorgonibm) [reply build/ci/jenkins_pipeline_build:6]: How much time it will add?

Comment (oriyarde) [reply build/ci/jenkins_pipeline_build:6]: currently less than 2 minutes

Comment (igorgonibm) [reply build/ci/jenkins_pipeline_build:6]: 2 min is not much. i prefer clear logs

**Pull request merged**